### ISSUE TITLE
Help, plz

### ISSUE THREAD

**Issue by Jockey721** [miracleyoo/Trainable-i3d-pytorch#1]
Did you reproduce the official PyTorch Flow predictions by using the optical flow made by self?? ?
I can't, and i find that the extracted dense flow of（https://github.com/qijiezhao/py-denseflow） is different from that of (https://github.com/yangwangx/denseFlow_gpu)

Following is my reproduced results:
RGB model
Top 5 classes and associated probabilities:
[playing cricket]: 9.999969E-01
[catching or throwing baseball]: 1.174602E-06
[playing kickball]: 5.298376E-07
[hitting baseball]: 3.268340E-07
[catching or throwing softball]: 2.987818E-07
Flow model
Top 5 classes and associated probabilities:
[playing trumpet]: 1.102161E-01
[playing clarinet]: 1.000090E-01
[playing trombone]: 8.973780E-02
[rock climbing]: 5.936522E-02
[cheerleading]: 5.848792E-02
===== Final predictions ====
logits proba class
2.532676e+01 9.999809e-01 playing cricket
1.271949e+01 3.347550e-06 skateboarding
1.246523e+01 2.595977e-06 celebrating
1.222022e+01 2.031856e-06 catching or throwing baseball
1.208718e+01 1.778761e-06 high kick

**Comment by Jockey721** [comment]
i confused the channels.
